lighthouse-core/report/html/report-styles.css

```diff
@@ -99,19 +101,22 @@
   --color-red: #FF4E42;
   --color-white: #FFFFFF;
   --header-padding: 20px 0 20px 0;
+  --icon-size: 24px;
   --score-container-padding: 12px;
   --score-container-width: 160px;
   --score-number-font-size-big: 42px;
   --score-number-font-size: 34px;
-  --circle-size-big: 120px;
-  --circle-size: 96px;
   --score-title-font-size-big: 28px;
   --score-title-font-size: 20px;
   --score-title-line-height-big: 36px;
   --score-title-line-height: 26px;
   --section-padding: 40px;
+  --topbar-bg: var(--color-black-100);
+  --topbar-height: 36px;
   --topbar-padding: 0 8px;
 
+  --logo-url: url('data:image/svg+xml;utf8,<svg height=192px version=1.1 viewBox="0 0 192 192"width=192px xmlns=http://www.w3.org/2000/svg xmlns:xlink=http://www.w3.org/1999/xlink><title>symbol_1_black</title><g id=symbol_1_black fill=none fill-rule=evenodd stroke=none stroke-width=1><g id=Group><g><rect height=192 id=Rectangle width=192 x=0 y=0></rect><path d="M67.7054588,179.352453 L70.3079591,158.532451 L119.643145,142.141282 L124.294541,179.352453 C115.41478,182.365787 105.898235,184 96,184 C86.1017646,184 76.5852201,182.365787 67.7054588,179.352453 L67.7054588,179.352453 Z M52.439998,172.48 C25.8942826,157.328 8,128.753717 8,96 C8,47.398942 47.398942,8 96,8 C144.601058,8 184,47.398942 184,96 C184,128.753717 166.105717,157.328 139.560002,172.48 L130,96 L136,96 L136,80 L128,80 L128,48 L96,28 L64,48 L64,80 L56,80 L56,96 L62,96 L52.4399988,172.48001 L52.439998,172.48 Z M113.875485,96 L116.756843,119.050869 L73.439198,133.48254 L78.1245155,96 L113.875485,96 Z M80,80 L80,56.8679623 L96,46.8679623 L112,56.8679623 L112,80 L80,80 Z"fill=#000000 id=Shape></path></g></g></g></svg>');
```

Comment:
```suggestion
  --logo-url: url('data:image/svg+xml;utf8,<svg height=192px version=1.1 viewBox="0 0 192 192"width=192px xmlns=http://www.w3.org/2000/svg xmlns:xlink=http://www.w3.org/1999/xlink><title>symbol_1_black</title><g id=symbol_1_black fill=none fill-rule=evenodd stroke=none stroke-width=1><g id=Group><g><rect height=192 id=Rectangle width=192 x=0 y=0></rect><path d="M67.7054588,179.352453 L70.3079591,158.532451 L119.643145,142.141282 L124.294541,179.352453 C115.41478,182.365787 105.898235,184 96,184 C86.1017646,184 76.5852201,182.365787 67.7054588,179.352453 L67.7054588,179.352453 Z M52.439998,172.48 C25.8942826,157.328 8,128.753717 8,96 C8,47.398942 47.398942,8 96,8 C144.601058,8 184,47.398942 184,96 C184,128.753717 166.105717,157.328 139.560002,172.48 L130,96 L136,96 L136,80 L128,80 L128,48 L96,28 L64,48 L64,80 L56,80 L56,96 L62,96 L52.4399988,172.48001 L52.439998,172.48 Z M113.875485,96 L116.756843,119.050869 L73.439198,133.48254 L78.1245155,96 L113.875485,96 Z M80,80 L80,56.8679623 L96,46.8679623 L112,56.8679623 L112,80 L80,80 Z"fill=%23000000 id=Shape></path></g></g></g></svg>');

```

Reply:
[replace the # with %23](https://stackoverflow.com/questions/54453987/did-chrome-72-break-dataimage-svgxmlutf8-for-css-background-svgs)

Reply:
I also added " around every prop.  I'm not sure if that's critical, but it works when I do that, but it might contain another change I forgot I made.